DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/filter.ts

```diff
@@ -132,7 +132,7 @@ function getSelectedFilterFromDropdownAndResetDropdown(
 
 function addNewFilterValueToFilterBy(newFilterValue: string): void {
   const filterBy = getFilterByValue();
-  if (!filterBy.includes(newFilterValue)) {
+  if (filterBy.split(filterSeparator).indexOf(newFilterValue) === -1) {

```

Comment:
Can we do `.split(filterSeparator).includes(newFilterValue)`?

Reply:
VS was giving me an error 😠